File: src/main/java/com/yerti/core/utils/NMSUtils.java
package com.yerti.core.utils;


import com.yerti.core.YertiPlugin;

public class NMSUtils {
    public static Class<?> getNMSClass(String name) {
        try {
            return Class.forName("net.minecraft.server." + YertiPlugin.getHookedPlugin().getServer().getVersion() + "." + name);
        } catch (ClassNotFoundException e) {
            e.printStackTrace();
            return null;
        }
    }
}

File: src/main/java/com/yerti/core/utils/NMSUtils.kt
package com.yerti.core.utils

import com.yerti.core.YertiPlugin.Companion.hookedPlugin

object NMSUtils {
    fun getNMSClass(name: String): Class<*>? {
        return try {
            Class.forName("net.minecraft.server." + hookedPlugin!!.server.version + "." + name)
        } catch (e: ClassNotFoundException) {
            e.printStackTrace()
            null
        }
    }
}